Add a movie by its title and date

josephschmitt opened the pull request:
Closes #11 

Adds a couple of new utterances so you can limit your movie results by the release date. You can now add a movie by saying:

> Add The Godfather released in 1972
> Add The Godfather from 1972
> Add The Godfather that came out in 1972
> Add The Godfather released in March 1972

josephschmitt committed:
Add release date support

hackworth reviewed (approved):
Users will need to update their Interaction Model, not sure how to get the word out, other than that it works great (along with the ES6 support)

josephschmitt commented:
@Hackworth good point. I'll note it in the commit message.

josephschmitt commented:
@Hackworth hey FYI I changed the intent schema a bit in my latest commit, so if you pull again, make sure to update the intent schema and utterances. I simplified it so that release date is an optional parameter of the main add movie intent instead of its own thing
